src/bfb_to_raw_img.sh

```diff
@@ -65,6 +67,11 @@ while [[ $# -gt 0 ]]; do
             shift # past argument
             shift # past value
             ;;
+        -os|--os)
```

Comment:
Maybe instead of adding an argument we can identify from bfb file the OS and running the relevant script inside docker.